housekeeper/store/file_filters.py

```diff
@@ -0,0 +1,38 @@
+from enum import Enum
+from typing import Callable, List, Optional
+from sqlalchemy.orm import Query
+from sqlalchemy import func as sqlalchemy_func
+
+from housekeeper.store.models import File, Tag
+
+
+def filter_files_by_id(files: Query, file_id: int, **kwargs) -> Query:
+    """Filter files by file id."""
+    return files.filter(File.id == file_id)
+
+
+def filter_files_by_tags(files: Query, tags: List[str], **kwargs) -> Query:
+    """Filter files by tags."""
+    files_tags = files.join(File.tags)
+    return files_tags.filter(Tag.name.in_(tags)).group_by(File.id).having(sqlalchemy_func.count(Tag.name) == len(tags))
```

Comment:
I think I prefer keeping the filter_files_by_tags in this module, keeping all of the logic for filtering files in one module.